docs/DeveloperGuide.md

```diff
@@ -651,14 +651,26 @@ be updated in the `Command` classes.
     * Pros: Easy to implement.
     * Cons: May cause confusion for users who are used to the current behaviour.
 
-### Validate the email format for `Person` objects
+### 6.Validate the email format for `Person` objects
 
 Currently, the app does not validate the email format for `Person` objects. This is not ideal as the email format should be validated to ensure that the email is in the correct format. In the future, we hope to validate the email format for `Person` objects.
 
-** Aspect: How to validate the email format for `Person` objects:**
+**Aspect: How to validate the email format for `Person` objects:**
 
 * Update the `Person` class to have a validation regex for the email field.
     * Pros: Easy to implement.
     * Cons: May cause confusion for users who are used to the current behaviour.
 
----
+### 7.Update the error message for `view c` for single client case 
+
+Currently, the `view c` command returns the error: `If you wish to view another client please return home by entering 'list' before viewing another client.` if there is only one client in the `Clients` list regardless of if the index is valid or invalid. This is not ideal as if there is only 1 `Client` in the list and user inputs `view c 1` it should
+not return an error.
+
+**Aspect: How to edit the error message to accommodate both scenarios**
+
+* Update the condition checking `execute()` method of the `ViewClientCommand` class to account for 2 scenarios
+  * Scenario 1: If user is already viewing that client and inputs the same index, in this case `1`
+  * Scenario 2: If there is only 1 client in the `Clients` list and user inputs that index, also by inputting `1`
+  * Pros: Easy to implement, clears up confusion of users when using the `view c` command and thinking that their inputs were valid.
+  * Cons: Additional variable has to be added in order to account for Scenario 2
```

Comment:
Could you do me a favour and help me indent the pros and cons like the rest of the section and add the line at the bottom? Thanks!